### ISSUE TITLE
npm coughs and rolls over.

### ISSUE THREAD

**Issue by chopraaa** [asapach/peerflix-server#94]
Ran this on a clean Debian install.

```
root@v0rt3x:~# npm install -g peerflix-server
npm WARN engine peerflix-server@0.1.1: wanted: {"node":">=0.12"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN deprecated bittorrent-dht@3.2.6: Critical security issue fixed in 5.1.3. All users should upgrade. (More info: https://github.com/feross/bittorrent-dht/issues/87)
npm WARN deprecated static-favicon@1.0.2: use serve-favicon module
npm WARN deprecated node-uuid@1.4.7: use uuid module instead
npm WARN engine hawk@0.10.2: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN engine boom@0.3.8: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN engine cryptiles@0.1.3: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN engine hoek@0.7.6: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN engine sntp@0.1.4: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
/usr/local/bin/peerflix-server -> /usr/local/lib/node_modules/peerflix-server/server/bin.js
peerflix-server@0.1.1 /usr/local/lib/node_modules/peerflix-server
├── range-parser@1.2.0
├── pump@1.0.2 (once@1.4.0, end-of-stream@1.1.0)
├── mkdirp@0.5.1 (minimist@0.0.8)
├── connect-multiparty@1.2.5 (on-finished@2.1.1, qs@2.2.5, type-is@1.5.7, multiparty@3.3.2)
├── lodash@2.4.2
├── read-torrent@1.3.0 (xtend@4.0.1, magnet-uri@2.0.1, request@2.16.6, parse-torrent@4.1.0)
├── socket.io@1.7.2 (object-assign@4.1.0, socket.io-adapter@0.5.0, has-binary@0.1.7, debug@2.3.3, socket.io-parser@2.3.1, engine.io@1.8.2, socket.io-client@1.7.2)
├── torrent-stream@0.18.1 (bitfield@0.1.0, thunky@0.1.0, hat@0.0.3, random-access-file@0.3.2, ip@0.3.3, mkdirp@0.3.5, magnet-uri@2.0.1, end-of-stream@0.1.5, ip-set@1.0.1, bncode@0.5.3, compact2string@1.4.0, bittorrent-dht@3.2.6, rimraf@2.5.4, peer-wire-swarm@0.9.2, bittorrent-tracker@2.12.1, parse-torrent@4.1.0)
├── express@3.5.3 (methods@0.1.0, merge-descriptors@0.0.2, cookie@0.1.2, debug@0.8.1, cookie-signature@1.0.3, fresh@0.2.2, range-parser@1.0.0, buffer-crc32@0.2.1, mkdirp@0.4.0, commander@1.3.2, send@0.3.0, connect@2.14.5)
└── fluent-ffmpeg@2.1.0 (which@1.2.12, async@2.1.4)
root@v0rt3x:~# peerflix-server
/usr/bin/env: node: No such file or directory
```

**Comment by asapach** [comment]
I think Debian and Ubuntu don't like `node` binary, because there's a [conflicting package](https://packages.debian.org/jessie/node) and instead use [`nodejs`](https://packages.debian.org/jessie/nodejs). You should either install [`nodejs-legacy`](https://packages.debian.org/jessie/nodejs-legacy) or PPA: https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions or

**Comment by chopraaa** [comment]
Coughed but didn't roll over this time.

```
root@v0rt3x:~# npm install -g peerflix-server
npm WARN engine peerflix-server@0.1.1: wanted: {"node":">=0.12"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN deprecated bittorrent-dht@3.2.6: Critical security issue fixed in 5.1.3. All users should upgrade. (More info: https://github.com/feross/bittorrent-dht/issues/87)
npm WARN deprecated static-favicon@1.0.2: use serve-favicon module
npm WARN deprecated node-uuid@1.4.7: use uuid module instead
npm WARN engine hawk@0.10.2: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN engine hoek@0.7.6: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN engine boom@0.3.8: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN engine sntp@0.1.4: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
npm WARN engine cryptiles@0.1.3: wanted: {"node":"0.8.x"} (current: {"node":"0.10.29","npm":"1.4.21"})
/usr/local/bin/peerflix-server -> /usr/local/lib/node_modules/peerflix-server/server/bin.js
peerflix-server@0.1.1 /usr/local/lib/node_modules/peerflix-server
├── range-parser@1.2.0
├── mkdirp@0.5.1 (minimist@0.0.8)
├── pump@1.0.2 (once@1.4.0, end-of-stream@1.1.0)
├── connect-multiparty@1.2.5 (on-finished@2.1.1, qs@2.2.5, type-is@1.5.7, multiparty@3.3.2)
├── lodash@2.4.2
├── read-torrent@1.3.0 (xtend@4.0.1, magnet-uri@2.0.1, request@2.16.6, parse-torrent@4.1.0)
├── socket.io@1.7.2 (object-assign@4.1.0, socket.io-adapter@0.5.0, has-binary@0.1.7, debug@2.3.3, socket.io-parser@2.3.1, engine.io@1.8.2, socket.io-client@1.7.2)
├── torrent-stream@0.18.1 (bitfield@0.1.0, thunky@0.1.0, hat@0.0.3, random-access-file@0.3.2, ip@0.3.3, mkdirp@0.3.5, magnet-uri@2.0.1, end-of-stream@0.1.5, ip-set@1.0.1, bncode@0.5.3, compact2string@1.4.0, bittorrent-dht@3.2.6, rimraf@2.5.4, peer-wire-swarm@0.9.2, bittorrent-tracker@2.12.1, parse-torrent@4.1.0)
├── express@3.5.3 (methods@0.1.0, merge-descriptors@0.0.2, cookie@0.1.2, debug@0.8.1, cookie-signature@1.0.3, fresh@0.2.2, range-parser@1.0.0, buffer-crc32@0.2.1, mkdirp@0.4.0, commander@1.3.2, send@0.3.0, connect@2.14.5)
└── fluent-ffmpeg@2.1.0 (which@1.2.12, async@2.1.4)
root@v0rt3x:~# peerflix-server
Listening on http://localhost:9000
```

Even though it's working, I'd rather have a clean setup.
